reinforcement_learning/examples/test_cpp/test_loop.cc

```diff
@@ -99,6 +100,10 @@ void test_loop::validity_loop(size_t thread_id)
     }
 
     test_inputs.log(thread_id, i, response, loggers[thread_id]);
+    
+    if (sleep_interval) {
```

Comment:
```suggestion
    if (sleep_interval > 0) {
```

I'm not a fan of this c-style bool check. I'd rather it be explicit that we are comparing an integer